src/main/scala/com/cognite/spark/datasource/AssetsRelation.scala

```diff
@@ -102,6 +102,8 @@ class AssetsRelation(config: RelationConfig)(val sqlContext: SQLContext)
     post(config, uri"${baseAssetsURL(config.project)}/update", updateAssetsItem)
   }
 
+  override def fieldsWithPushdownFilter: Seq[String] = Seq("name", "source", "depth", "description")
```

Comment:
Do we need this to be a def? Seems a constant `val` would suffice.

Reply:
Good catch.